api/organisations/management/commands/createinitialorganisation.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import logging
+from typing import Any
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from organisations.models import Organisation, OrganisationRole
+from users.services import get_initial_superuser
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument(
+            "--email",
+            type=str,
+            dest="admin_email",
+            help="Email address for the superuser",
+            default=None,
+        )
+        parser.add_argument(
+            "--organisation-name",
+            type=str,
+            dest="organisation_name",
+            help="Organisation name",
+            default=None,
+        )
+
+    def handle(
+        self,
+        *args: Any,
+        admin_email: str | None,
+        organisation_name: str | None,
```

Comment:
It's strange to me that we allow no email and default to admin@example.com I take it this command is for people that want to spin something up really fast and use it right away?

Reply:
Yes. The PR does add a `bootstrap` call to the entrypoint so the folks spinning up a new instance are going to get these right away, unless `ALLOW_ADMIN_INITIATION_VIA_CLI` is set to `false`.